db/views_table.go

```diff
@@ -127,5 +131,57 @@ func ReadViewTable(ctx context.Context, tableName string) ([]api.ViewRow, error)
 		viewRows = append(viewRows, viewRow)
 	}
 
-	return viewRows, nil
+	return convertViewRecordsToNativeTypes(viewRows, columnDef), nil
+}
+
+// convertViewRecordsToNativeTypes converts view cell to native go types
+func convertViewRecordsToNativeTypes(viewRows []api.ViewRow, columnDef []api.ViewColumnDef) []api.ViewRow {
+	for _, viewRow := range viewRows {
+		for i, colDef := range columnDef {
+			if i >= len(viewRow) {
+				continue
+			}
+
+			if viewRow[i] == nil {
+				continue
+			}
+
+			switch colDef.Type {
+			case api.ViewColumnTypeGauge:
+				if raw, ok := viewRow[i].([]uint8); ok {
+					viewRow[i] = json.RawMessage(raw)
+				}
+
+			case api.ViewColumnTypeDuration:
+				switch v := viewRow[i].(type) {
+				case int:
+					viewRow[i] = time.Duration(v)
+				case int32:
+					viewRow[i] = time.Duration(v)
+				case int64:
+					viewRow[i] = time.Duration(v)
+				case float64:
+					viewRow[i] = time.Duration(int64(v))
+				default:
+					logger.Warnf("postProcessViewRows: unknown duration type: %T", v)
```

Comment:
I think we should return these as errors - they will pollute the logs and hide it from the user